tests/tests_unit/test_rules/test_models/test_entities.py

```diff
@@ -24,6 +25,36 @@
 
 
 TEST_CASES = [
+    (
+        ContainerIndexEntity,
+        "name",
+        ContainerIndexEntity(suffix="name"),
+    ),
+    (
+        ContainerIndexEntity,
+        "name(cursorable=True)",
+        ContainerIndexEntity(suffix="name", cursorable=False),
+    ),
```

Comment:
I don't think this test works. I ran it locally, and no matter what I changed cursorable to, the test did not fail. Only if I changed name, it would fail.

Reply:
To trigger a failure, you have to set strict to True in the call to _load in line 164.

I suggest you also add a non-happy-path test to verify this.

Reply:
Added test, found the bug, fixed it, and updated the test data.